src/App.vue

```diff
@@ -155,6 +164,31 @@ export default class App extends Mixins(
   /** The Update Current JS Date timer id. */
   private updateCurrentJsDateId = 0
 
+  get axios (): any {
+    return axios
+  }
+
+  get webChatReason (): string {
+    return window['webChatReason']
+  }
+
+  get webChatStatusUrl (): string {
+    return window['webChatStatusUrl']
+  }
+
+  get webChatUrl (): string {
+    return window['webChatUrl']
+  }
+
+  get openTooltipMessage (): String {
+    return 'Click here to chat live with Helpdesk staff about Name Requests.'
+  }
+
+  get closedTooltipMessage (): String {
+    return 'We are closed. The Service BC Contact Centre is open Monday through Friday' +
+           '7:30am - 5:00pm Pacific Time excluding BC statutory holidays.'
+  }
```

Comment:
Nice use of getters to provide messages 👍 

You don't need the other getters. You can simply do like this: https://github.com/bcgov/business-create-ui/blob/25827cb57a6bae5089ff60dd79fb553cbf6e6a2f/src/App.vue#L76

And this: https://github.com/bcgov/business-create-ui/blob/25827cb57a6bae5089ff60dd79fb553cbf6e6a2f/src/App.vue#L337

Reply:
I think you don't need the closed tooltip message -- the default message in the shared component looks the same to me.

If you added the message here to make sure it's separate from the default message (in case someone changes the default) then that's OK with me.

Reply:
I removed the closed tooltip message. You are right, it is not necessary.